src/key_variants/single_key.rs

```diff
@@ -11,7 +11,9 @@ use canonical::Canon;
 #[cfg(feature = "canon")]
 use canonical_derive::Canon;
 use dusk_bls12_381::BlsScalar;
```

Comment:
We have warnings on this section of the code that should be removed. This part should be shielded by the `std` feature

Reply:
Solved!